<launch>

  <!-- set /global_example parameter -->  
  <group ns="search_port">

    <!--set /climb_pkg_para/close all 3dof driver-->
    <param name="close_all_3dof_climb_driver_flag" value="1"/>

    <!--set /climb_pkg_para/enable_control_stand_bar 1 enable 2 disable-->
    <param name="enable_control_stand_bar" value="0" />
    <!--set /climb_pkg_para/enable_control_rotation 1 enable 2 disable-->
    <param name="enable_control_rotation" value="1"/>
    <!--set /climb_pkg_para/enable_climb_control-->
    <param name="enable_climb_control" value="0"/>

    <!--set /climb_pkg_para/enable_control_stand_bar-->
    <param name="velocity_control_stand_bar" value="1000" />
    <!--set /climb_pkg_para/enable_control_rotation-->
    <param name="velocity_control_rotation" value="2000"/>
    <!--set /climb_pkg_para/enable_climb_control-->
    <param name="velocity_climb_control" value="1000"/>

    <!--set /climb_pkg_para/enable_control_stand_bar neg up pos down-->
    <param name="distance_control_stand_bar" value="-15.0" />
    <!--set /climb_pkg_para/enable_control_rotation-->
    <param name="rad_control_rotation" value="-4.0"/>
    <!--set /climb_pkg_para/enable_climb_control neg up pos down-->
    <param name="distance_climb_control" value="-15.0"/>    

    <node pkg="painting_robot_demo" type="climb_node.py" name="climb_node" output="screen" >
    </node>
    
  </group>
  
</launch>

